concent_api/common/decorators.py

```diff
@@ -184,6 +185,19 @@ def wrapper(
             **kwargs: dict,
         ) -> Union[HttpRequest, JsonResponse]:
             assert database_name in settings.DATABASES or database_name is None
+
+            if not if_given_version_of_golem_messages_is_compatible_with_version_in_concent(
+                request=request,
+                client_public_key=client_public_key
+            ):
+                serialized_message = dump(
+                    message.concents.ServiceRefused(
+                        reason=message.concents.ServiceRefused.REASON.InvalidRequest,
```

Comment:
I agree but instead of leaving `#TODO` comment please add issue in `concent` repository on github and make sure it is added in pivotal with blocker set to the issue created in `golem-messages`.

Reply:
I tried to create PR, but I don't have permission to golem_messages repository.
@dybi @rwrzesien could you add this? I suggest:
`UnsupportedProtocolVersion = 'UNSUPPORTED_PROTOCOL_VERSION'`

Reply:
Issue added:
https://github.com/golemfactory/golem-messages/issues/275